packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx

```diff
@@ -28,6 +37,48 @@ export interface ActionButtonsProps {
   className?: string;
 }
 
+const AnimatedUpvoteIcons = React.memo(function InnerAnimatedUpvoteIcons() {
```

Comment:
I think we can actually move this to a custom file.
Then have the file use the https://storybook.daily.dev/?path=/docs/experiments-helpers--docs#withexperiment-hoc method to either return UpvotedButton (if part of experiment) or fallback to regular button.
That way it's cleaner in this file and all logic will live in one file.

Plus easier to manage enrollment and eventually cleanup.

Reply:
Done ✅